pkg/controller/kogitoapp/resource/build_config_s2i.go

```diff
@@ -38,21 +40,6 @@ const (
 	mavenMirrorURLEnvVar         = "MAVEN_MIRROR_URL"
 )
 
-var (
-	// DefaultBuildS2IJVMCPULimit is the default CPU limit for JVM s2i builds
-	DefaultBuildS2IJVMCPULimit = v1alpha1.ResourceMap{Resource: v1alpha1.ResourceCPU, Value: "2"}
```

Comment:
why removing it?
No default values needed anymore or it comes from other place?

Reply:
There's no default anymore, should have been removed years ago. :)
Setting resources requirements affects how pods are prioritized by the cluster, that's why we left this up to the user.

Reply:
They are used by this [test](https://github.com/kiegroup/kogito-cloud-operator/pull/203/files#diff-5dace150bc5d66edc6ebb1ade36bc6b2) once only. I don't think we use these default values anywhere else.